environments/staging-phase-two/group_vars/all/vars.yml

```diff
@@ -45,7 +45,7 @@ nfs_server_for_specials: staging-files00.cnx.org
 
 nfs_connections: "{{ vault_nfs_connections }}"
 
-webview_version: v0.21.0
```

Comment:
No, but it should be updated to what's in production, which is currently `v0.21.11-dev7fefc3d`. How a development version hit production is beyond me.